examples/simple_ckks_bootstrapping.jl

```diff
@@ -23,7 +23,22 @@ function simple_ckks_bootstrapping(context)
     sv_after = bootstrap!(sv)
 
     result = decrypt(sv, private_key)
-    println("Output after bootstrapping \n\t", result)
+    println("Output vector after bootstrapping \n\t", result)
+
+    ################################################################################    
+    m = [0.25 0.5 0.75; 1.0 2.0 3.0; 4.0 5.0 6.0]
```

Comment:
```suggestion
    m = [0.25 0.50 0.75;
         1.00 2.00 3.00;
         4.00 5.00 6.00]
```
Maybe for increase readability?